jwst/pipeline/calwebb_spec2.py

```diff
@@ -292,12 +290,14 @@ def process_exposure_product(
 
             # Call the resample_spec step for 2D slit data
```

Comment:
This comment needs to be updated, because the next step called now is pixel_replace, not resample_spec.

Reply:
Done